ecosystem/typescript/sdk/src/transaction_builder/serde/binaryDeserializer.ts

```diff
@@ -0,0 +1,126 @@
+import { Deserializer } from "./deserializer";
+
+export abstract class BinaryDeserializer implements Deserializer {
```

Comment:
I don't know if we need to define an interface for a one-off. I might argue against it altogether.

Reply:
Fair enough.